src/core/prodfactor.jl

```diff
@@ -210,6 +209,10 @@ function prodfactor(wind::Wind, latitude::Real, longitude::Real, time_steps_per_
         @ccall hdl.ssc_data_set_array(wind_resource::Ptr{Cvoid}, "fields"::Cstring, 
             fields::Ptr{Cdouble}, length(fields)::Cint)::Cvoid
 
+        if size(resources)[1] == 1
```

Comment:
@rathod-b will you please add a comment in the code here explaining why we need this `if` statement?

Reply:
@rathod-b @NLaws should we consider this resolved?